user-guide.rst

```diff
@@ -4,24 +4,28 @@ User Guide & FAQ
 Frequently Asked Questions
 --------------------------
 
-.. rubric:: Why are there different Scenario Explorer for specific projects?
+.. rubric:: Why are there different *Scenario Explorer* instances for specific projects?
 
 Each project usually brings together different consortium partners and uses a
 project-specific list of variable and regions (see :ref:`project-templates`).
 
 It is therefore easier to keep projects separate, with its own database **platform**,
 *Scenario Explorer* instance and related GitHub repository (see :ref:`scenario-processing`).
 
-.. rubric:: What is the level of spatial granularity?
+.. rubric:: What is the level of spatial granularity of scenarios?
 
-The spatial granularity of the data depends on the scope of the project and the
+The spatial granularity of the scenario data depends on the scope of the project and the
 contributing modeling teams. The IAMC data format supports both aggregated data as well
 as more detailed regional data like country-level resolution (see :ref:`countries`)
 or the :ref:`nuts` used by the European Union.
 
 .. rubric:: Which sectors are covered by the variable template?
 
-The :ref:`variables` are flexible and can be adapted to the needs of the project.
+The sectoral coverage of the data format is flexible and can be adapted to the needs of
+the project. Read more about the :ref:`variables`.
+
+You can also look at the `common-definitions`_ repository for a list of variables
```

Comment:
Since this is aimed at beginner users, linking to the GitHub directory without much further explanation might be confusing. 
I'd suggest to (also) link to the Excel version of common-definitions.

Reply:
The xlsx link is highlight in the common-definitions Readme, and I'd rather users are a bit confused when looking at a GitHub repository than directly downloading an xlsx file without knowing that it is coming from a larger ecosystem.